scripts/StressTest.ps1

```diff
@@ -0,0 +1,57 @@
+﻿<#
+.SYNOPSIS
+    Stress test a BinSkim build.
+.PARAMETER BinSkimFolder
+    The path to a folder where we would find the BinSkim executable.
+.PARAMETER OutputFileName
+    The name that will be used to output the SARIF log file.
+.PARAMETER InputPaths
+    The paths that will be used as argument in BinSkim.
+#>
+
+[CmdletBinding()]
+param(
+    [string]
+    $BinSkimFolder = "..\bld\bin\x64_Release\netcoreapp3.1",
+
+    [string]
+    $OutputFileName = "binskim-hash-run-",
+
+    [string]
+    $InputPaths
```

Comment:
If you add
`[Parameter(Mandatory=$true)]`
this should block looping on an error because of no input path.

Reply:
updated. thanks